### DISCUSSION TITLE
Google Translate prevents Bootstrap tabs to work

### DISCUSSION THREAD

**Discussion by dejudicibus** [twbs/bootstrap#35777] [Q&A]
When I use Google Translate, Bootstrap tabs do not work anymore. For example, this is an original page of mine in Italian language:

https://genealogia.dejudicibus.it/index.php?f=dejudicibus&b=napoli&c=xviii&p=1785a

and this is the same page translated to English

https://genealogia-dejudicibus-it.translate.goog/index.php?f=dejudicibus&b=napoli&c=xviii&p=1785a&_x_tr_sl=it&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp

As you can see, if you click on tabs, for example "Portraits",, nothing happens. Any idea why?

**Comment by florianlacreuse** [comment]
Google Translate changes the `href` value of `.nav-link` items.

**Comment by dejudicibus** [comment]
OK, so... how can I fix that?

On Thu, Feb 3, 2022 at 1:52 PM Florian Lacreuse ***@***.***>
wrote:

> Google Translate changes the href value of .nav-link items.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/twbs/bootstrap/discussions/35777#discussioncomment-2103529>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAK5PEARSUE7VHBRVUM7ENTUZJ3CFANCNFSM5NOP5OQQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Dario de Judicibus, Rome, Italy (EU)
Site: https://www.dejudicibus.it, https://genealogia.dejudicibus.it
Blog: https://www.lindipendente.eu
Book: https://www.lalamanera.it, https://www.lasorgentedeimondi.it

**Reply by florianlacreuse** [reply]
Sorry I don't know, IMHO it doesn't seem to be a Bootstrap related issue.
